inst/NEWS.Rd

```diff
@@ -9,6 +9,8 @@
     \itemize{
       \item Safer \code{Rcpp_list*}, \code{Rcpp_lang*} and
       \code{Function.operator()} (Romain in \ghpr{1014}, \ghit{1015}).
+      \item A number of \code{#nocov} markers were added (Dirk in
+      \ghprP1036}). 
```

Comment:
```suggestion
      \ghpr{1036}). 
```

Reply:
Hmpf. I could swear I can a check which should spotted that.  Will fix!